sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/README.md

```diff
@@ -97,6 +97,20 @@ if __name__ == '__main__':
     loop.run_until_complete(main())
 ```
 
+#### Use `BlobCheckpointStore` with a different version of Azure Storage Service API
+Some environments have different versions of Azure Storage Service API. For instance, some Azure Stack use 2017-11-09.  
+`BlobCheckpointStore` by defaults uses the Storage Service API version 2019-07-07. To use it against a different
+ version, specify that version number when you create the `BlobCheckpointStore` object.
+ 
+```python
+from azure.eventhub.extensions.checkpointstoreblobaio import BlobCheckpointStore
+checkpoint_store = BlobCheckpointStore.from_connection_string(
+        storage_connection_str,
+        container_name,
+        headers={"x-ms-version": "2017-11-09"}
```

Comment:
In theory we could still expose this as an `api_version` parameter here - but then internally convert it to the `headers` parameter when creating the ContainerClient.

Reply:
Added back "api_version" support